<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPerson</class>
 <widget class="QDialog" name="DlgPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="title">
        <string>Персональные данные</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leLastName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leName"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leMidName"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Должность</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="leProf"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Дата рождения</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDateEdit" name="deBirth">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Дата приема</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDateEdit" name="deStart">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>СНИЛС</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="leSNILS">
          <property name="inputMask">
           <string>000-000-000 00</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Класс условий труда</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="leSpecClass"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Дата проведения</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QDateEdit" name="deSpecClass">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox">
       <property name="title">
        <string>Планирование</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabMed">
           <attribute name="title">
            <string>Медосмотр</string>
           </attribute>
           <widget class="QListWidget" name="listWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>200</y>
              <width>471</width>
              <height>171</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>170</y>
              <width>100</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Вредные факторы</string>
            </property>
           </widget>
           <widget class="QGroupBox" name="formGroupBox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>211</width>
              <height>129</height>
             </rect>
            </property>
            <property name="title">
             <string>Медосмотр</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="pushButton_5">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Дата проведения</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Период</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Следущий</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="formGroupBox_2">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>20</y>
              <width>241</width>
              <height>131</height>
             </rect>
            </property>
            <property name="title">
             <string>Психиатрическое освидетельствование</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="2" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Дата проведения</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Период</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Следующее</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_6">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_7">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="pushButton_6">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>129</x>
              <y>167</y>
              <width>31</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>168</x>
              <y>167</y>
              <width>31</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tabOT">
           <attribute name="title">
            <string>Обучение по ОТ</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabPB">
           <attribute name="title">
            <string>Обучение по ПБ</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabPrB">
           <attribute name="title">
            <string>Промышленная безопасность</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgPerson</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgPerson</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
